notebooks/demo_processes.ipynb

```diff
@@ -428,21 +436,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "/home/slim/thunderbird/tests/metadata-conversion/simple-conversion.yaml\n",
       "update_metadataResponse(\n",
-      "    output='http://localhost:5001/outputs/790e526a-b66e-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
+      "    output='http://localhost:5001/outputs/010c4f42-b6ff-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
       ")\n"
      ]
     }
    ],
    "source": [
     "updates = resource_filename(\"tests\", \"metadata-conversion/simple-conversion.yaml\")\n",
+    "print(updates)\n",
```

Comment:
Probably want to delete the `print` statement. Checking the filepath is no longer our interest.